packages/worker/src/connection-options.ts

```diff
@@ -6,19 +6,14 @@ type TLSConfig = native.TLSConfig;
 
 export { TLSConfig };
 
-export interface ServerOptions {
+export interface NativeConnectionOptions {
```

Comment:
Would it be helpful at all for the two connection options extend a generic interface?

```
export sharedConnectionOptions: GenericConnectionOptions = { address: ... };
```

Versus:

```
export sharedConnectionOptions = { address: ... };
```